### DISCUSSION TITLE
Start menu icon

### DISCUSSION THREAD

**Discussion by ajshovon** [valinet/ExplorerPatcher#472] [Ideas]
It would be nice to have a option to select Windows 10's or Windows 11's start menu icon.

**Comment by valinet** [comment]
There are a dozen similar posts by now. It’s a work in progress. Someone suggested how to tackle it, it’s discussed in this forum but currently progress on this is stagnating: https://github.com/valinet/ExplorerPatcher/discussions/436. Once the issues are sorted out, it will ship, of course.
